src/app/app-platform/ContentApp.cpp

```diff
@@ -41,23 +41,22 @@ namespace AppPlatform {
 #define ZCL_DESCRIPTOR_CLUSTER_REVISION (1u)
 #define ZCL_APPLICATION_BASIC_CLUSTER_REVISION (1u)
 
-EmberAfStatus ContentApp::HandleReadAttribute(ClusterId clusterId, AttributeId attributeId, uint8_t * buffer,
-                                              uint16_t maxReadLength)
+Status ContentApp::HandleReadAttribute(ClusterId clusterId, AttributeId attributeId, uint8_t * buffer, uint16_t maxReadLength)
```

Comment:
This file needs to actually pull in Status instead of relying on namespace pollution by headers.